### PR TITLE
Breaking

### PR DESCRIPTION
**Pull request by rowanmanning**

### CODE REVIEW

File: MIGRATION.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+
+# Migration
+
+## Migrating from v3 to v4
+
+### Mixins
+
+- The `oTooltips` mixin now includes all styles required for tooltips. See the [Sass documentation](README.md#sass) for information on the new parameters
```
Comment (notlee) [inline MIGRATION.md]:
```suggestion
- The `oTooltip` mixin now includes all styles required for tooltips. See the [Sass documentation](README.md#sass) for information on the new parameters
```

**Comment by origamiserviceuser** [comment]
o-tooltip bundle size difference from 3.5.1 to 1072a0fbfc7d9c3135dfc79f248bc7a090a72bdc
js: 0.46kb increase (0.19kb/gzip)
An insignificant difference was also found for: css (master), css (internal), css (whitelabel)

**Comment by origamiserviceuser** [comment]
o-tooltip bundle size difference from 3.5.1 to bb05bc3a514dba2e605690398fe0ae8cc375b8c9
css, master: 11.31kb increase (0.85kb/gzip)
css, internal: 11.31kb increase (0.85kb/gzip)
css, whitelabel: 11.3kb increase (0.85kb/gzip)
An insignificant difference was also found for: js

**Comment by origamiserviceuser** [comment]
o-tooltip bundle size difference from 3.5.1 to 087159d78504b7fc0836f3a5553d6e5a188511ae
css, master: 11.31kb increase (0.85kb/gzip)
css, internal: 11.31kb increase (0.85kb/gzip)
css, whitelabel: 11.3kb increase (0.85kb/gzip)
An insignificant difference was also found for: js

**Comment by origamiserviceuser** [comment]
o-tooltip bundle size difference from 3.5.1 to cb1ca46c97327ab1e9fbe632e1ef99e5265cc85a
css, master: 11.2kb increase (0.82kb/gzip)
css, internal: 11.2kb increase (0.83kb/gzip)
css, whitelabel: 11.19kb increase (0.82kb/gzip)
An insignificant difference was also found for: js

**Pull request merged**